ESQL: Validate unique plan attribute names (#110488)
elastic/elasticsearch: docs/reference/esql/processing-commands/grok.asciidoc

`pattern`::
A grok pattern.

> In case a field name coincides with an existing column, the existing column is discarded.

Reviewer: ```suggestion
If a field name conflicts with an existing column, the existing column is discarded.
```

|===
include::{esql-specs}/docs.csv-spec[tag=grokWithToDatetime-result]
|===

> In case a field name is used more than once, `GROK` creates a multi-valued

Reviewer: ```suggestion
If a field name is used more than once, `GROK` creates a multi-valued
```
